### ISSUE TITLE
Missing :empty suffix for lexicons

### ISSUE THREAD

**Issue by pand5461** [KSP-KOS/KOS#2589]
It's strange that lexicons have the `:length()` suffix but don't have the `:empty()` suffix.

**Comment by Dunbaratu** [comment]
I suppose it could be added.  It's not strictly needed since checking ``:length = 0`` works just fine, but it could be a few less operations to just return the boolean directly instead of doing the comparison.

If implemented, it wouldn't be just Lexicon.  It would be all the container types: LIST, QUEUE, etc.  Because anywhere that has a ``:LENGTH`` suffix, the ``:empty`` suffix would have an obvious meaning.

**Comment by pand5461** [comment]
I agree the checks `:length() = 0` or `:keys():empty()` are valid workarounds.

It just seems inconsistent that all other containers have that suffix and it appears that computing the length of a lexicon is not required on the C# level to check if it's empty.

**Comment by jonnyboyC** [comment]
It appears that all the collection class derive from `EnumerableValue<T, TE>` which already has the  `:empty()` suffix. So list, stack, queue, and uniqueset would already have it. Obviously lexicon could just implement this `:empty()` suffix, but any thoughts on simply making lexicon enumerable?

I had poked around a bit and see that the `:case` seems to make that hard as the comparer isn't settable and the innerEnumerable isn't settable either.
